amgut/db/patches/0009.sql

```diff
@@ -0,0 +1,229 @@
+--Create pet survey in database
```

Comment:
this patch number conflicts with #428 i believe, but just making a note as its of course dependent on merge order